content/en/blog/2020/addon-rework/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Reworking our Addon Integrations
+description: A new way to manage installation of telemetry addons.
+publishdate: 2020-06-10
+attribution: John Howard (Google)
+keywords: [telemetry,addons,integrations,grafana,prometheus]
+test: n/a
+---
+
+Starting with Istio 1.6, we are introducing a new method for integration with telemetry addons (Grafana, Prometheus, Zipkin, Jaeger, Kiali, etc).
+
+In previous releases, these addons were bundled as part of the Istio installation. At the time, this was helpful as it allowed users to quickly get started without needing any complicated configurations to install and integrate with these applications. However, it came with some issues:
+
+* Our installations were not as up to date or feature rich as upstream installation methods, meaning users were left missing out on some of the great features provided by these applications, for example:
+    1. persistent storage
+    1. features like `Alertmanager` for Prometheus
+    1. advanced security settings
+*  Integration with existing deployments that were using these features was more challenging than it should be.
```

Comment:
what does "features" refer to here?

Reply:
Refers to the previous bullet point. Basically what I mean to say here is that users were using their own deployments already, because they needed advanced customization/organization guidelines/existing integrations. When they were doing that, it was more challenging that it should be. so I guess we can probably just make this something like " Integration with existing deployments was more challenging than it should be."

Reply:
`... more challenging than it should have been.`